core/controllers/editor_test.py

```diff
@@ -557,6 +557,23 @@ def test_exploration_download_handler_for_default_exploration(self):
         # Check downloaded dict.
         self.assertEqual(self.SAMPLE_JSON_CONTENT, response)
 
+        # Check downloading a specific version.
+        download_url = (
+            '/createhandler/download/%s?output_format=%s&width=50&v=1' %
```

Comment:
Should we keep the width parameter in the tests? (on line 553 and 570 as well)

Reply:
Removed. Thanks for catching this!